docs/specifications/methods/README.md

```diff
@@ -0,0 +1,9 @@
+![Zonemaster](https://github.com/dotse/zonemaster/blob/master/docs/images/zonemaster_logo_black.png)
+==========
+
+### Purpose of this directory
+
+This contents of this directory includes contains other specification, currently only 
```

Comment:
"This contents of this directory includes" -> "This directory"?
"specification" -> "specifications"?

Reply:
Will fix.